modular_skyrat/modules/modular_items/lewd_items/code/technical_stuff_for_lewd.dm

```diff
@@ -83,6 +83,23 @@
 		/obj/item/polepack = 1)
 	generate_items_inside(items_inside,src)
 
+//Shibari stand
+/obj/item/storage/box/shibari_stand
+	name = "DIY Shibari stand kit"
+	desc = "Contains everything you need to build your own shibari stand!"
+
+/obj/item/storage/box/shibari_stand/PopulateContents()
+	var/static/items_inside = list(
+		/obj/item/shibari_stand_kit = 1,
+		/obj/item/wrench = 1,
+		/obj/item/screwdriver= 1,
```

Comment:
Take the tools out of this. I already removed all the tools from the other vendor contents because they were being used to give prisoners almost everything they needed to break out of jail. This is half of what they need to hack a door.

Reply:
Absolutely reasonable, we’ll delete them